aat/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you git-ignore this one as well?

Reply:
.project files are needed for eclipse-based IDEs, and it is good to include them in the repo so that it takes much less time to setup the work environment with .projects files ready. Otherwise, the developer/tester will have to establish the work environment with lots of .project file manually generated again and again.

Reply:
I won't reject the PR but won't accept it either. I feel we should not commit IDE related config files. In fact I believe we should keep the clutter as small as possible and adding lots of .project files will increase it. We should not mandate this to be "use whatever IDE you want and add whatever config you want".